### PR TITLE
proposed update to the blog

### PR DESCRIPTION
**Pull request by linsun**
Link to IBM's blog - https://developer.ibm.com/blogs/istio-google-open-usage-commons/

Clarify we at the end so it is not confused.

[ ] Configuration Infrastructure
[x ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

@louiscryan @dcberg @rvennam PTAL and let me know if this is a good place to link to it.

### CODE REVIEW

File: content/en/blog/2020/open-usage/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Since [day one](/news/releases/0.x/announcing-0.1/), the Istio project has belie
...
+As one of the founders of the project, Google is the current owner of the Istio trademark. While anyone who is using the software in accordance with the license can use the trademarks, the historic ownership has caused some confusion and uncertainty about who can use the name and how, and at times this confusion has been a barrier to community growth. So today, as part of Istio’s continued commitment to openness, Google is announcing that the Istio trademarks will be transferred to a new organization, the Open Usage Commons, to provide neutral, independent oversight of the marks, while we understand there are [concerns with the the new organization from IBM](https://developer.ibm.com/blogs/istio-google-open-usage-commons/).
```
Comment (rvennam) [inline content/en/blog/2020/open-usage/index.md:13]:
```suggestion
As one of the founders of the project, Google is the current owner of the Istio trademark. While anyone who is using the software in accordance with the license can use the trademarks, the historic ownership has caused some confusion and uncertainty about who can use the name and how, and at times this confusion has been a barrier to community growth. So today, as part of Istio’s continued commitment to openness, Google is announcing that the Istio trademarks will be transferred to a new organization, the Open Usage Commons, to provide neutral, independent oversight of the marks, while we understand there are [concerns with the new organization from IBM](https://developer.ibm.com/blogs/istio-google-open-usage-commons/).
```

Comment (ssuchter) [inline content/en/blog/2020/open-usage/index.md:13]: Hi Lin - Let's discuss this on Friday in the Steering meeting.

Comment (rvennam) [reply content/en/blog/2020/open-usage/index.md:13]: Consider moving to a new sentence.
`We understand there are [concerns with the new organization from IBM](https://developer.ibm.com/blogs/istio-google-open-usage-commons/) and others in the community.`

Comment (dcberg) [reply content/en/blog/2020/open-usage/index.md]: Agree that it would read better as a separate sentence.

**Comment by linsun** [comment]
closing it, Sean will rework the blog to clarify the "we".

**Comment by ssuchter** [comment]
PTAL: https://github.com/istio/istio.io/pull/7706

**Pull request closed without merging**